front/src/interfaces/atoms/Avatar/IAvatar.ts

```diff
@@ -0,0 +1,7 @@
+import React, { MouseEvent } from 'react';
```

Comment:
```suggestion
import { MouseEvent } from 'react';
```

React를 사용하지 않는다는 eslint waring이 있습니다.